### DISCUSSION TITLE
Query by linked records via REST API [Jun 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2404] [Discord Threads]
> 🙋‍♂️ **Cris Acosta**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@724687678511251489> :: for a scenario of country 'has many' city relation, you wish to query city information using link present in the country table? is my understanding correct?   
####    
#### you can give us an example otherwise.  
#  
> 🙋‍♂️ **Cris Acosta**  
#### How filter for Categoria´s Id?   
####    
#### {   
####   "list": [   
#### {   
####       "CodAtendLazer": 2,   
####       "StatusAtendimento": "5",   
####       "ModificadoEm": "2022-04-01 11:09:03",   
####       "DestinosRead": {   
####         "CodDestino": 6,   
####         "Destino": "Aracaju"   
####       },   
####       "Agente": "Guilherme Rocha",   
####       "FaseVendaRead": {   
####         "CodFase": 18,   
####         "FaseVenda": "Atualização de orçamento"   
####       },   
####       "ClientesRead": {   
####         "CodCliente": 2,   
####         "IdTipoPessoa": 0   
####       },   
####       "Nome": "Aline Marques Labcom",   
####       "Categoria": {   
####         "Id": 2,   
####         "Nome": "Venda",   
####         "Comentario": null,   
####         "CreatedAt": "2022-06-04 09:32:23",   
####         "UpdatedAt": "2022-06-04 09:48:25",   
####         "CriadoPor": 1   
####       }   
####     }   
#### }  
#  
> 👨‍🦲 **dstala**  
#### <@724687678511251489> Currently, you can create filter based on 'primary value' of a linked record [primary value is the field content that gets displayed on creating a link]. You can configure primary value field by using API/ in UI.  
#  
> 🙋‍♂️ **Cris Acosta**  
#### Hi <@857998597474484295> !   
####    
#### I make a field with this.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983566986351095918)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
